pmda/test/test_rmsf.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+import MDAnalysis as mda
+import numpy as np
+import os
+import pytest
+import pmda
+import pmda.rmsf
+from MDAnalysis.analysis.rms import RMSF
+from numpy.testing import assert_almost_equal
+from MDAnalysisTests.datafiles import GRO_MEMPROT, XTC_MEMPROT
+
+
+@pytest.fixture(scope='module')
+def u():
+    return mda.Universe(GRO_MEMPROT, XTC_MEMPROT)
+
+
+def test_rmsf_sum(u):
+    PMDA = pmda.rmsf.RMSF(u.atoms)
+    core_number = 1
+    PMDA.run(n_blocks=core_number, n_jobs=core_number)
+    MDA = mda.analysis.rms.RMSF(u.atoms).run()
+    assert np.sum(MDA.rmsf) == np.sum(PMDA.rmsf)
+
+
+def test_rmsf_values(u):
+    PMDA = pmda.rmsf.RMSF(u.atoms)
+    core_number = 1
```

Comment:
Eventually write tests for different core numbers; see e.g. https://github.com/MDAnalysis/pmda/blob/943ed2b4996d704f93bb279451df90dcbe5ebaa2/pmda/test/test_parallel.py#L72

Reply:
You should probably also test for a few higher `n_jobs` such as 3, 4.